src/main/java/seedu/address/model/client/Client.java

```diff
@@ -87,6 +89,15 @@ public LastModifiedInstant getLastModifiedInstant() {
         return Collections.unmodifiableSet(this.clientNotes);
     }
 
+    /**
+     * Gets the list of client notes associated with this client as an observable arraylist.
+     *
+     * @return An Observable List of client notes associated with this client.
```

Comment:
Javadoc here should be unmodifiable? Also, can you change "arraylist" to "ArrayList"?

Reply:
resolved